openprescribing/pipeline/metadata/tasks.json

```diff
@@ -277,6 +277,14 @@
             "upload_to_bigquery"
         ]
     },
+    "create_importlog_for_measures": {
+        "type": "post_process",
+        "command": "create_importlog_for_measures",
+        "dependencies": [
+            "import_measures",
+            "generate_ppu"
```

Comment:
Why is `ppu` a dependency for a task that marks measure calculation as complete?  I don't suppose it matters much, but just want to check I understand the logic.

Reply:
The short answer is that `measures` is actually not the right name but I couldn't easily think of a better one at the time. We need both the measures and PPU to finish importing before we can produce an updated dashboard. But now it occurs to me that `dashboard_data` might be a reasonable name.